tests/shared/src/test/scala/epollcat/TcpSuite.scala

```diff
@@ -173,6 +173,18 @@ class TcpSuite extends EpollcatSuite {
       .interceptMessage[BindException]("Address already in use")
   }
 
+  test("BindException - EADDRNOTAVAIL") {
+    // 240.0.0.1 is in reserved range 240.0.0.0/24.  Used to elicit Exception.
+    IOServerSocketChannel
+      .open
+      .use { ch =>
+        for {
+          _ <- ch.bind(new InetSocketAddress("240.0.0.1", 0))
+        } yield ()
+      }
+      .interceptMessage[BindException]("Cannot assign requested address")
```

Comment:
@LeeTibbert why not do this?
```suggestion
      .interceptMessage[BindException](
        if (isLinux) "Cannot assign requested address"
        else "Can't assign requested address"
      )
```

Reply:
Thank you for the suggestion.  I did not know that was possible. Learning new things is good,
but also an exercise in humility.

The code snippet works fine on SN, after importing  LinktimeInfo.

Understandably, it fails on JVM.  May need to factor this test out and
use the fact that epollcat is a multi-project to have  a new Suite
of OS specific tests.  I __hate__ having duplicate code, as it
is hard to keep in sync.    So, worst case, I/we can fix this.

I am off to look around to see how the test might determine
the OS it is running on.  (perhaps more complicate code 
within the intercept message to read the System Property
"os.name")   Can the code (thunk?) have more than one
statement, as long as it returns a String?)

Reply:
You can add this definition at the top of the suite:
```scala
val isLinux = {
  val osName = Option(System.getProperty("os.name"))
  osName.exists(_.toLowerCase().contains("linux"))
}
```